src/RealtimeEffectPanel.cpp

```diff
@@ -1213,6 +1301,8 @@ void RealtimeEffectPanel::SetTrack(AudacityProject& project, const std::shared_p
          : mToggleEffects->PopUp();
       mEffectList->SetTrack(project, track);
       mProject = &project;
+      //i18n-hint: argument - track name
+      mHeader->SetName(wxString::Format(_("Realtime effects for %s"), track->GetName()));
```

Comment:
If language is changed in preferences, this line is not visited to update the name.